src/lib/network/network_autoyast.rb

```diff
@@ -48,49 +51,30 @@ def merge_configs(conf)
     end
 
     # Creates udev rules according definition from profile
-    #
-    # FIXME: Currently used only for applying udev rules during network
-    # installations (ssh, vnc, ...). It was introduced as a quick fix for
-    # bnc#944349, so it is currently limited only on {ssh|vnc} installations.
     def create_udevs
```

Comment:
It seems this method is not covered with unit tests. Is it tested in another way?